apps/server/src/services/dictionaryService.ts

```diff
@@ -211,3 +212,8 @@ export const updateSchema = async (id: string, schema: Schema, major: boolean):
 	// Save new dictionary version
 	return await DictionaryRepo.addDictionary(omit(updatedDictionary, '_id'));
 };
+
+export const downloadDictionaryByNameAndVersion = async (name: string, version: string) => {
+	const dict = await getOneByNameAndVersion(name, version);
+	return replaceReferences(dict);
```

Comment:
What does this function do different from `getOneByNameAndVersion`? Is it just the reference replacement? It's not actually providing a download or anything like that.